WordPress.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -0,0 +1,16 @@
+{
+  "object": {
+    "pins": [
+      {
+        "package": "ScreenObject",
+        "repositoryURL": "https://github.com/Automattic/ScreenObject",
+        "state": {
+          "branch": "trunk",
+          "revision": "201ace5a0e5eb7ed2005a34763041b7022625f43",
+          "version": null
+        }
+      }
+    ]
+  },
+  "version": 1
+}
```

Comment:
My Xcode generates this file, but I seems redundant given the packages info is already in the repo.

I added it just to show it to you and see if you know more. In Simplenote we decided not to add it.

It _might_ be due to the fact that I'm still on Catalina (:flush: – This week I'll update. Promise)

Reply:
While working on the WooCommerce version of this experiment, I found what I think is [the right answer to the question](https://github.com/woocommerce/woocommerce-ios/pull/4392#discussion_r648001032) of whether tracking this file is appropriate.

Yes it is

> This looks to me like the `Podfile.lock` or `package-lock.json` equivalent for the Xcode managed dependency resolution. The Xcode project is the source of truth for the configuration and this is the snapshot of the dependencies as resolved from that configuration that the project expects.